### DISCUSSION TITLE
Attraction: A GIS based game

### DISCUSSION THREAD

**Discussion by Anemnox** [jakobzhao/geog495#70] [Final Project]
Application: https://anemnox.github.io/GIS-Game/
Github: https://github.com/anemnox/GIS-Game
Description: A start of a little simulation game based on the school boundaries in the WA area. This map displays some generated "resources" that are randomly generated based on the boundary polygons. As of now the map generates a new board and a set of randomly generated resources. Hovering over some of the boundary areas, reveals all the resources in that area and a small description of the attributes to the area
<img width="1462" alt="Screen Shot 2021-12-09 at 11 59 54 PM" src="https://user-images.githubusercontent.com/23712257/145538718-4818a849-97bb-4e87-ab30-a8702be1a343.png">

**Comment by ebar23** [comment]
I think that the concept behind your map is very intuitive and cool. I never thought about being able to gamify GIS work but you seemed to have made it happen! I will say that I was a little lost when interacting with your app. I wasn't exactly sure of the direction or information I should be getting out of it. This might just be me and my inability to use your app correctly, but I might suggest that if you decide to go the game route, that you include an instructions pop-up as they do in most games to be able to assist your users in their performance. Overall, I think that this is a pretty cool map and idea and I wish you the best moving forward :)

**Comment by lucretiachen99** [comment]
I really like the idea of making this into a simulation game and the overall aesthetics look very cool. However, something that could improve would be to label the areas better because it is currently very hard to navigate where each place is other than looking at the water bodies and a few borders on the map. It would help to add in the title of each location directly onto the map. Overall the map is very well made!
